pkg/ebpf/tracer_windows.go

```diff
@@ -45,7 +171,46 @@ func (t *Tracer) getConnections(active []ConnectionStats) ([]ConnectionStats, ui
 
 // GetStats returns a map of statistics about the current tracer's internal state
 func (t *Tracer) GetStats() (map[string]interface{}, error) {
-	return nil, ErrNotImplemented
+	if t.driverHandle == syscall.InvalidHandle {
+		return nil, fmt.Errorf("Problem with handle cannot get stats.")
+	}
+
+	var (
+		bytesReturned uint32
+		stats         C.struct_driver_stats
+		statbuf       = make([]byte, C.sizeof_struct_driver_stats)
+		ioctlcd       = ctl_code(NETWORK_DEVICE_TYPE_CTL_CODE, DDFILTER_IOCTL_GETSTATS, uint32(0), uint32(0))
+	)
+
+	err := syscall.DeviceIoControl(t.driverHandle, ioctlcd, nil, 0, &statbuf[0], uint32(len(statbuf)), &bytesReturned, nil)
```

Comment:
If we can get the stats object from other calls that will occur in the pulling of packets, it might be worth updating the variables through that method rather than making a separate call here.

Currently, the expvarStats() function means that we will be hitting the driver every 5 seconds with a call. I don't know if that means we might want to open up another, specific handle on the driver for pulling stats.

Reply:
I don't think a call every 5 seconds is going to cause performance issues unless the stats call does some locking under the hood.

Reply:
I believe it does do some locking. The `DeviceIoControl` function ultimately calls syscall9 which locks the OS thread:

- https://github.com/golang/go/blob/master/src/runtime/syscall_windows.go#L205-L216

My original thought here was somewhat related to my other comment below that we'd only ever have stats for this handle, especially if we make it the "stats pulling handle". 

In light of this being a locking call, though, should I decrease the rate at which we call this, or look into reporting stats by some other method?

Reply:
I'm less worried about the OS thread blocking and more worried about if a call to stats will interfere with ReadFile.